cycler.py

```diff
@@ -124,6 +124,46 @@ def keys(self):
         """
         return set(self._keys)
 
+    def key_change(self, old, new):
```

Comment:
What? You don't do music?  Eh, it probably does read better that way, and I can stop getting Whitney Houston songs playing in my head every time I read this method...